beta/tests/tensorflow/data/configs/sequential_model_cifar10_rb_sparsity.json

```diff
@@ -0,0 +1,40 @@
+{
+    "model": "SequentialModel",
+    "pretrained": false,
+    "input_info": {
+        "sample_size": [1, 32, 32, 3]
+    },
+    "num_classes": 10,
+
+    "optimizer": {
+        "type": "adam",
```

Comment:
I think it's not valid because this [PR](https://github.com/openvinotoolkit/nncf/pull/531) was merged. Can you please check **all configs in your PR** and update them?

Reply:
Done